src/ddb/README.md

```diff
@@ -51,154 +49,59 @@
 
+Help Options:
+  -h, --help        Show this help message
 ```
 
+Interactive mode help
 ```
-$ ddb -R help
-help
-	Show help message for all the commands.
-
-quit
-	Quit interactive mode
-
-ls [path]
-	List containers, objects, dkeys, akeys, and values
-    [path]
-	Optional, list contents of the provided path
-Options:
-    -r, --recursive
-	Recursively list the contents of the path
-
-open <path>
-	Opens the vos file at <path>
-    <path>
-	The path to the vos file to open. This should be an absolute
-	path to the pool shard. Part of the path is used to
-	determine what the pool uuid is.
-Options:
-    -w, --write_mode
-	Open the vos file in write mode. This allows for modifying
-	the vos file with the load, commit_ilog, etc commands.
-
-close
-	Close the currently opened vos pool shard
-
-dump_superblock
-	Dump the pool superblock information
-
-dump_value <path> <dst>
-	Dump a value to a file
-    <path>
-	The vos tree path to dump. Should be a complete path
-	including the akey and if the value is an array value it
-	should include the extent.
-    <dst>
-	The file path to dump the value to.
-
-rm <path>
-	Remove a branch of the VOS tree. The branch can be anything
-	from a container and everything under it, to a single value.
-    <path>
-	The vos tree path to remove.
-
-load <src> <dst>
-	Load a value to a vos path. This can be used to update the
-	value of an existing key, or create a new key.
-    <src>
-	The source file path that contains the data for the value to
-	load.
-    <dst>
-	The destination vos tree path to the value where the data
-	will be loaded. If the path currently exists, then the
-	destination path must match the value type, meaning, if the
-	value type is an array, then the path must include the recx,
-	otherwise, it must not.
-
-dump_ilog <path>
-	Dump the ilog
-    <path>
-	The vos tree path to an object, dkey, or akey.
-
-commit_ilog <path>
-	Process the ilog
-    <path>
-	The vos tree path to an object, dkey, or akey.
-
-rm_ilog <path>
-	Remove all the ilog entries
-    <path>
-	The vos tree path to an object, dkey, or akey.
-
-dump_dtx <path>
-	Dump the dtx tables
-    <path>
-	The vos tree path to a container.
-Options:
-    -a, --active
-	Only dump entries from the active table
-    -c, --committed
-	Only dump entries from the committed table
-
-clear_cmt_dtx <path>
-	Clear the dtx committed table
-    <path>
-	The vos tree path to a container.
-
-smd_sync
-	Restore the SMD file with backup from blob
+$ help
+
+The DAOS Debug Tool (ddb) allows a user to navigate through and modify 
+a file in the VOS format. In order to modify the file, the '-w' option must 
+be included when opening the vos file.
 
+Many of the commands take a vos tree path. The format for this path
+is 'cont_uuid/obj_id/dkey/akey/recx'. The keys currently only support string 
+keys. The recx for array values is the format {lo-hi}. To make it easier to 
+navigate the tree, indexes can be used instead of the path part. The index 
```

Comment:
```suggestion
is 'cont_uuid/obj_id/dkey/akey/recx'. The keys currently only support string
keys. The recx for array values is the format {lo-hi}. To make it easier to
navigate the tree, indexes can be used instead of the path part. The index
```